### DISCUSSION TITLE
Failed to fetch /api/auth/session - CLIENT_FETCH_ERROR

### DISCUSSION THREAD

**Discussion by TorbynNare** [nextauthjs/next-auth#5712] [Help] [answered]
Hello everyone, I'm new to next-auth and this is my first discussion post here. Looking forward to being apart of the community!

When cloning the [next-auth-example](https://github.com/nextauthjs/next-auth-example) repo on my personal windows machine, everything works perfectly.

However, on my M1 Mac, when cloning the exact same repo, I'm met with the following errors...

![Screen Shot 2022-11-02 at 12 03 50 PM](https://user-images.githubusercontent.com/21169571/199579091-b2f917ba-71ea-404b-b16d-d65fb2f7e216.jpg)

The redirects seem to be bouncing between 'session' & 'session/' as shown below until it errors...

![Screen Shot 2022-11-02 at 12 07 27 PM](https://user-images.githubusercontent.com/21169571/199579748-32da60da-cc9a-4719-a2fe-dca2ae51d0f0.jpg)

----

I'm aware that [CLIENT_FETCH_ERROR](https://next-auth.js.org/errors#client_fetch_error) is normally resolved by setting a [NEXTAUTH_URL](https://next-auth.js.org/configuration/options#nextauth_url) .env variable, which I have. Here is my .env.local

`NEXTAUTH_URL=http://localhost:3000`
`NEXTAUTH_SECRET=<insert-randomly-generated-hex32-value>`

I've tried setting `NEXTAUTH_URL=http://127.0.0.1:3000` and the same error occurs.

When attempting to Sign In, I'm redirected to `http://localhost:3000/api/auth/error`  where only the following is displayed.

![Screen Shot 2022-11-02 at 12 22 57 PM](https://user-images.githubusercontent.com/21169571/199582523-0407ecd3-e7c2-4898-bfde-9f0297b75e51.jpg)

Any thoughts would be greatly appreciated!

----

**System**:
- OS: macOS 12.6
- CPU: (8) arm64 Apple M1 Pro
- Memory: 105.63 MB / 16.00 GB
- Shell: 5.8.1 - /bin/zsh

**Binaries**:
-  Node: 16.18.0 - /usr/local/bin/node
-  Yarn: 1.22.19 - /opt/homebrew/bin/yarn
-  npm: 8.19.2 - /opt/homebrew/bin/npm

**Browsers**:
-  Chrome: 107.0.5304.87
-  Safari: 16.1

**npmPackages**:
-  next: 12.2.5 => 12.2.5 
-  next-auth: 4.15.0 => 4.15.0 
-  react: ^18.2.0 => 18.2.0

**Comment by TorbynNare** [comment, marked as answer]
Super odd, however, after re-installing node w/ brew, deleting all generated project files, and giving the mac a fresh restart things seem to be working now. I guess the typical, "turn it off/on again" was the case here.

No real conclusion as to what was causing this issue, but nonetheless, it's resolved. 🤷🏽

**Reply by NorkzYT** [reply]
@TorbynNare 

I have the same issue, I have done all that you have said, any ideas?

Here is my issue.
https://github.com/nextauthjs/next-auth/issues/5805
